README.md

```diff
@@ -22,26 +22,31 @@ JAVA&nbsp;7 when working with this project.
 
 ## GETTING STARTED
 
+  There is a single python script build.py that is used to build and run
+  the project. There are many ways to run a python script. The syntax
+  below works on linux if the script is in your current directory and set
+  to executable (e.g., cd codeu_project_2017; chmod +x build.py)
```

Comment:
As students need to run this on Windows, we should avoid unix specific instructions. I agree with @crepricg , using `python build.py` would be easier for students to use.